ios/FluentUI/Notification/FluentNotification.swift

```diff
@@ -259,10 +264,15 @@ public struct FluentNotification: View, ConfigurableTokenizedControl {
             HStack(spacing: tokens.horizontalSpacing) {
                 image
                 textContainer
-                Spacer(minLength: tokens.horizontalPadding)
+                if !isFlexibleWidthToast {
+                    Spacer()
```

Comment:
Did we want to add a new token for this spacer, rather than removing the token usage entirely? Or is there another way to customize the space between the text and button?

Reply:
Ah I meant to follow up on this--I removed the minimum length of Space because I think it results in something like this:

|-image-horizontalSpacing-textContainer-**horizontalSpacing-Spacer of min Length-horizontalSpacing**-button-|

which is creating way too large of a space. Though even when the Spacer is removed, the gap between textContainer and button is still larger than the horizontal spacing and I can't figure out why.

Reply:
I just did some testing with a quick demo app and addressed this!

Reply:
In swiftui, what's the best way to make sure notfiicationview has the highest content hugging enforced but the text container has the compression resistance to make this behave correctly?